src/Microsoft.Health.Dicom.Core/Features/Common/IServerCredentialProvider.cs

```diff
@@ -8,23 +8,23 @@
 using System.Threading.Tasks;
 using Azure.Core;
 
-namespace Microsoft.Health.Dicom.Core.Features.Export;
+namespace Microsoft.Health.Dicom.Core.Features.Common;
 
 /// <summary>
-/// Represents a provider for identities used by the export operation.
+/// Represents a credential provider for the identity used by the server.
 /// </summary>
-public interface IExportIdentityProvider
+public interface IServerCredentialProvider
```

Comment:
I tried to come up with a new name that is somehow separate from the internal managed identity lol